src/App.js

```diff
@@ -1,7 +1,42 @@
+import { Routes, Route, useLocation } from "react-router-dom";
+import { Signup, Login, Home, Explore, Bookmark, Profile } from "pages";
+import { Navbar, SuggestionSidebar } from "components";
+import { Box, Center, Container } from "@chakra-ui/react";
 import "./App.css";
 
 function App() {
-  return <div className="App"></div>;
+  const { pathname } = useLocation();
+  const publicRoutePathnames = ["/login", "/signup", "/"];
```

Comment:
I think you forgot to remove this "/login"

Reply:
thanks for pointing out